src/frontend/src/btc/components/send/BtcSendReview.svelte

```diff
@@ -14,14 +17,17 @@
 	export let amount: number | undefined = undefined;
 	export let networkId: NetworkId | undefined = undefined;
 	export let source: string;
+	export let utxosFee: UtxosFee | undefined = undefined;
 
 	let hasPendingTransactionsStore: Readable<BtcPendingSentTransactionsStatus>;
 	$: hasPendingTransactionsStore = initPendingSentTransactionsStatus(source);
 
 	let disableSend: boolean;
-	// We want to disable send if pending transactions are loading, there was an error or there are pending transactions.
+	// We want to disable send if pending transactions or UTXOs fee are loading, there was an error or there are pending transactions.
 	$: disableSend =
-		$hasPendingTransactionsStore !== BtcPendingSentTransactionsStatus.NONE || invalid;
+		$hasPendingTransactionsStore !== BtcPendingSentTransactionsStatus.NONE ||
+		isNullish(utxosFee) ||
```

Comment:
The comments says "when UTXOs fee are loading" but, the if says "when utxos are not defined or null". Is that correct?